walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -8,16 +8,98 @@
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either exptimeBetweens or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
 package com.google.sps;
 
+import java.util.Set;
+import java.util.HashSet;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+
+        //this gets the list of optional attendees for the requested meeting
```

Comment:
Change from "this" to "This". The comments should be complete sentences. Also, add a punctuation "." at the end of the sentence.